physionet-django/project/models.py

```diff
@@ -2627,3 +2627,17 @@ def check_passphrase(self, raw_passphrase):
 
         return isnot_expired and check_password(raw_passphrase, self.passphrase)
 
+
+class Metrics(models.Model):
+    '''
+    Stores the daily and running total view counts for each project, as well
+    as date recorded.
+    '''
+    core_project = models.ForeignKey('project.CoreProject', 
+                                     on_delete=models.DO_NOTHING, 
+                                     default=None)
+    slug = models.SlugField(max_length=MAX_PROJECT_SLUG_LENGTH, db_index=True,
```

Comment:
Is the slug field needed? Is it a worry that it could potentially get out of sync with the project slugs?

Reply:
Upon further thought, the slug field is not needed, so I took it out.